aula-css/exercicios/exercicio 1 css/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+    <header class="container">
+        <div class="perfil">
+            <div>
+                <img id="perfil" src="imagem/cat.jpg" alt="foto de perfil é do gato">
+            </div>
+            <div>
+                <p><h1>Leila´s Photo Grid</h1></p>
+                <p><h2>I´m a woman from São Paulo, Brazil.</h2></p>
+            </div>
+        </div>
+    </header>
+
+    <section class="container">
+        <ul class="imagens">
+          
+            <li class="fotos">
+                <a href="detalhes-1.html">
+                <img src="imagem/gato na arvore2.jpg" alt="Gato na árvore">
```

Comment:
```suggestion
                <img src="imagem/gato na arvore2.jpg" alt="Gato na árvore" />
```

Fique atenta à barrinha no final deste elemento `<img>`, somente os elementos que não possuem tag de fechamento são escritos dessa forma, outro exemplo familiar seria o `<br/>`